controlpanel/core/models/user.py

```diff
@@ -0,0 +1,28 @@
+from controlpanel.core.common.utils import sanitize_dns_label
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    user_id = models.CharField(max_length=128, primary_key=True)
+    name = models.CharField(max_length=256, blank=True)
+    nickname = models.CharField(max_length=256, blank=True)
+
+    REQUIRED_FIELDS = ["email", "user_id"]
+
+    class Meta:
+        db_table = "control_panel_user"
+        ordering = ("nickname",)
```

Comment:
If `nickname` can be blank, then not sure about using it for ordering, as it will put blank values first

Reply:
let me change it to `name`